libpod/runtime.go

```diff
@@ -364,13 +364,17 @@ func makeRuntime(ctx context.Context, runtime *Runtime) (err error) {
 	// Make lookup tables for runtime support
 	supportsJSON := make(map[string]bool)
 	supportsNoCgroups := make(map[string]bool)
+	supportsKVM := make(map[string]bool)
 	for _, r := range runtime.config.Engine.RuntimeSupportsJSON {
 		supportsJSON[r] = true
 	}
 	for _, r := range runtime.config.Engine.RuntimeSupportsNoCgroups {
 		supportsNoCgroups[r] = true
 	}
-
+	/*	for _, r := range runtime.config.Engine.RuntimeSupportsKVM {
```

Comment:
What's going on here?

Reply:
Oops had not implemented, it is now in containers/common.